contrib/objective-C/KalmanFilter.m

```diff
@@ -0,0 +1,77 @@
+/**
+ * Simple implementation of the Kalman Filter for 1D data.
+ * Originally written in JavaScript by Wouter Bulten
+ *
+ * Now rewritten into Objective-C
+ * 2018
+ *
+ * @license MIT License
+ *
+ * @author Sifan Ye
+ * @author Andreas Eppler
+ * @author Biraj Dhakal
+ *
+ * @see https://github.com/wouterbulten/kalmanjs
+ *
+ */
+
+#import <Foundation/Foundation.h>
+
+@interface SampleClass:NSObject
+/* method declaration */
+- (double)filter:(double)measurement;
+@end
+
+@implementation SampleClass
+double cov=NAN;
+double x=NAN;
+- (double)filter:(double)measurement {
```

Comment:
Could you add support for the control here? (u variable)

Reply:
Hi @bidh. No problem, let my try to explain it further: The input to the kalman filter is a combination of the measurement and the control (u variable). In many cases u is not used as it is not known or hard to estimate. However, when known it is important to add this to the filter.

Please see the javascript implementation here: https://github.com/wouterbulten/kalmanjs/blob/master/src/kalman.js#L40

The change is probably simple: Instead of the constant u, change that to a variable with a default of 0.